subsys/bluetooth/Kconfig

```diff
@@ -131,6 +131,26 @@ config BT_EXT_ADV_MAX_ADV_SET
 	  Maximum number of simultaneous Bluetooth advertising sets
 	  supported.
 
+config BT_PER_ADV
+	bool "Periodic Advertising and Scanning support [EXPERIMENTAL]"
+	help
```

Comment:
depends on BT_EXT_ADV

Reply:
It's already positioned within the `if BT_EXT_ADV` clause. I could pull it out and place it outside and add the `depends on` if that's preferable?

Reply:
I missed the `if BT_EXT_ADV`